components/FormFlashcard/FormFlashcard.js

```diff
@@ -11,12 +13,47 @@ export default function FormFlashcard({
   currentFlashcard,
   onSubmitFlashcard,
   isFormClosing,
+  changeCollections,
 }) {
+  const [showNewCollectionFields, setShowNewCollectionFields] = useState(false);
+
+  function handleCollectionChange(event) {
+    if (event.target.value === "newCollection") {
+      setShowNewCollectionFields(true);
+    } else {
+      setShowNewCollectionFields(false);
+    }
+  }
+
+  function addNewCollection(formData) {
```

Comment:
I would recommend reconsidering the function names. The collection is actually added in the `_app.js`, so the function should be called `addNewCollection`, `handleAddNewCollection`, or something similar, instead of `changeCollections`. Looking at your other functions in the `_app.js`, I would suggest `handleAddCollection` as it would be consistent with the naming of the existing functions.

Reply:
Of course, then you will need to find another name for the function in line 28. 🙂